gemhardware/include/gem/hw/GEMHwDevice.h

```diff
@@ -581,10 +525,45 @@ namespace gem {
        * @brief Performs basic setup for the device
        * sets connection details (OBSOLETE)
        * sets logging level to Error
-       * Not inherited
+       * Not inherited, but calls a pure virtual function...
```

Comment:
Someone reading the documentation will most likely wonder which virtual function.